src/components/forms/CreateInvoiceForm.js

```diff
@@ -0,0 +1,295 @@
+import React, { useContext } from 'react';
+import { motion } from 'framer-motion';
+import { FieldArray, Formik } from 'formik';
+
+import { InvoiceContext } from '../../context/InvoiceContext';
+import { initialFormValues, validationSchema } from './data';
+import Input from './Input';
+import Terms from './Terms';
+import ItemList from './ItemList';
+import DatePicker from './DatePicker';
+import SimpleBar from 'simplebar-react';
+import { genId } from '../../functions';
+import arrowLeft from '../../images/icon-arrow-left.svg';
+import 'simplebar/dist/simplebar.min.css';
+
+const CreateInvoiceForm = ({ setShowForm }) => {
+    const invoiceContext = useContext(InvoiceContext);
+    const { createInvoice } = invoiceContext;
+
+    const saveInvoice = (data) => {
```

Comment:
Nema potrebe da se deklarise kao konstanta kada je po prirodi funkcija
```suggestion
    function saveInvoice(data) {
```